romancal/saturation/saturation.py

```diff
@@ -34,15 +34,15 @@ def flag_saturation(input_model, ref_model):
         the GROUPDQ array
     """
 
-    data = input_model.data[np.newaxis, :]
+    data = input_model.data.value[np.newaxis, :]
 
     # Create the output model as a copy of the input
     output_model = input_model
     gdq = output_model.groupdq[np.newaxis, :]
     pdq = output_model.pixeldq[np.newaxis, :]
 
     # Copy information from saturation reference file
-    sat_thresh = ref_model.data.copy()
+    sat_thresh = ref_model.data.value
```

Comment:
There's a change in behavior here; do you want .value.copy() ?

Reply:
Changed.